Drive/rover_drive_system.hpp

```diff
@@ -351,23 +363,16 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
     units::angle::degree_t right_wheel_angle =
         angle;  // Needs to be set by server
     double x_angle = static_cast<double>(right_wheel_angle);
```

Comment:
```suggestion
    double x_angle = static_cast<double>(right_wheel_angle);
```
x_angle is not very helpful. What is x_angle? Maybe calling it lead_wheel or right wheel might make more sense?